src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -32,7 +32,7 @@ public static class DataTestUtility
         public static readonly string AKVClientSecret = null;
         public static List<string> AEConnStrings = new List<string>();
         public static List<string> AEConnStringsSetup = new List<string>();
-        public static readonly bool EnclaveEnabled = false;
+        public static bool EnclaveEnabled = false;
```

Comment:
Why this change? I don't see other changes where the value is being updated and similar variables right below are still readonly. So wondering...

Reply:
reverted.